Unify desc format for SVG elements (#36883)
mdn/content: files/en-us/web/svg/element/textpath/index.md

{{SVGRef}}

> To render text along the shape of a {{SVGElement("path")}}, enclose the text in a **`<textPath>`** [SVG](/en-US/docs/Web/SVG) element that has an {{SVGAttr("href")}} attribute with a reference to the {{SVGElement("path")}} element.

Reviewer: This one doesn't look very consistent. Can it be reworded?

> To render text along the shape of a {{SVGElement("path")}}, enclose the text in a **`<textPath>`** [SVG](/en-US/docs/Web/SVG) element that has an {{SVGAttr("href")}} attribute with a reference to the {{SVGElement("path")}} element.

Author: ```suggestion
The **`<textPath>`** [SVG](/en-US/docs/Web/SVG) element that has an {{SVGAttr("href")}} attribute with a reference to the {{SVGElement("path")}} element renders text that enclosed in this element along the shape of a {{SVGElement("path")}}
```

> The **`<textPath>`** [SVG](/en-US/docs/Web/SVG) element that has an {{SVGAttr("href")}} attribute with a reference to the {{SVGElement("path")}} element renders text that enclosed in this element along the shape of a {{SVGElement("path")}}

Reviewer: ```suggestion
The **`<textPath>`** [SVG](/en-US/docs/Web/SVG) element is used to render text along the shape of a {{SVGElement("path")}} element.
The text must be enclosed in the `<textPath>` element and its {{SVGAttr("href")}} attribute is used to reference the desired `<path>`.
```
